### PR TITLE
Feat: TablePress importer

### PR DESCRIPTION
**Pull request by kariae**
This PR introduces a bulk tables importer to TablePress. It accepts CSV and JSON files from a given folder.

It uses the filename as the table name and the first row of data as the table header (same as the GUI importer of TablePress).

Since the import command uses the internal classes/methods of the plugin TablePress, the plugin should be installed before using it.

Below some examples to use the command line:

```bash
# Import bulk CSV tables from /tmp folder.
wp newspack-content-migrator import-table-press-tables --input-dir=/tmp
# Import bulk JSON tables from /tmp folder.
wp newspack-content-migrator import-table-press-tables --input-dir=/tmp --input-type=json
```

The code introduces a new migrator logic class so it needs a review 🙇

### CODE REVIEW

File: src/MigrationLogic/TablePress.php [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+		$tablepress_import                    = $plugin_path . '/tablepress/classes/class-import.php';
+		$tablepress_controller                = $plugin_path . '/tablepress/classes/class-controller.php';
+		$tablepress_controller_admin          = $plugin_path . '/tablepress/controllers/controller-admin.php';
+		$included_tablepress                  = is_file( $tablepress ) && include_once $tablepress;
+		$included_tablepress_import           = is_file( $tablepress_import ) && include_once $tablepress_import;
+		$included_tablepress_controller       = is_file( $tablepress_controller ) && include_once $tablepress_controller;
+		$included_tablepress_controller_admin = is_file( $tablepress_controller_admin ) && include_once $tablepress_controller_admin;
+
+		if ( false === $included_tablepress || false === $included_tablepress_import || false === $included_tablepress_controller_admin || false === $included_tablepress_controller ) {
+			// TablePress is a dependency, and will have to be installed before the public functions/commands can be used.
```
Comment (iuravic) [inline src/MigrationLogic/TablePress.php:39]: This comment explains everything 👍  Similar to [the comment I left](https://github.com/Automattic/newspack-custom-content-migrator/pull/144#pullrequestreview-911650339) for the NinjaPress exporter PR, could we also add a validation check to the `cmd_import_table_press` method to check if the dependency is installed?

File: src/MigrationLogic/TablePress.php [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+	 * Import a table.
+	 *
+	 * @param string $format Import format.
+	 * @param string $data   Data to import.
+	 * @return array|false Table array on success, false on error.
+	 */
+	public function import_table( $format, $data, $filename, $description ) {
+		$reflector = new \ReflectionObject( $this->tablepress_controller_admin );
+		$importer  = $reflector->getProperty( 'importer' );
+		$importer->setAccessible( true );
```
Comment (iuravic) [inline src/MigrationLogic/TablePress.php:58]: Behold the power of PHP :)

**Review by iuravic — CHANGES_REQUESTED** [review]

File: src/MigrationLogic/TablePress.php (hunk 1 above)
Comment (kariae) [reply src/MigrationLogic/TablePress.php:39]: Thanks @iuravic for the quick review and for noticing this 👍

Comment (kariae) [reply src/MigrationLogic/TablePress.php:39]: Thanks @iuravic for the quick review and for noticing this 👍

**Review by iuravic — APPROVED** [review]

**Pull request merged**